broken-site-reporting/tests.json

```diff
@@ -20,6 +20,10 @@
                     {"name": "blockedTrackers", "value": "bad.tracker.test,tracking.test"},
                     {"name": "surrogates", "value": "surrogate.domain.test"}
                 ],
+                "urlParametersRemoved": "0",
```

Comment:
Readme says that those were meant to be booleans?

Reply:
Fair point :D Actually it depends on how you define a boolean, is it true/false or 1/0 or is it the same thing? The reporting failed because we were using true/false and the expectation was 1/0 so I guess is not really a boolean but a value of 1 or 0.